public/assets/modal-5c074a179fe73aecaae781027ff968de.js

```diff
@@ -0,0 +1 @@
+var hideModal=function(){$("#submit-button").click(function(){$(".modal-content").remove()})};
```

Comment:
Line is too long.<br>Missing semicolon.<br>'hideModal' is defined but never used.